flocker/node/agents/blockdevice.py

```diff
@@ -591,6 +622,8 @@ def run(self, deployer):
         return succeed(None)
 
 
+# Get rid of this in favor of calculating each individual operation in
+# BlockDeviceDeployer.calculate_changes.  FLOC-1771
```

Comment:
We may want to represent those other states to the end user anyway.  They might reveal interesting information about the state of their cluster.